java/performance/src/org/netbeans/modules/performance/guitracker/ActionTracker.java

```diff
@@ -758,11 +758,11 @@ public void exportAsXML(Document style, PrintStream out)
     }
 
     private static String getShortenName(String name) {
-        name = name.replaceAll("javax.swing", "j");
-        name = name.replaceAll("org.netbeans.modules", "o.n.m");
-        name = name.replaceAll("org.netbeans", "o.n");
-        name = name.replaceAll("org.openide.awt", "o.o.a");
-        name = name.replaceAll("org.openide", "o.o");
+        name = name.replace("javax.swing", "j");
```

Comment:
(Not completely equivalent again, but this time the new code is probably more correct than the old one.)

Reply:
Now, the string "javax.swing" is exactly matched instead of "java<any_character_here>swing"